docs/admin/licensing/index.mdx

```diff
@@ -1,18 +1,24 @@
 # Licensing
 
-Sourcegraph Enterprise requires a valid license key to enable many Enterprise-specific features.
+Sourcegraph requires a valid license key to enable many of its more prominent features.
 
-Sourcegraph will periodically perform a license validation check by contacting sourcegraph.com. This check sends no information other than a unique site ID and information about the configured Sourcegraph license. This check is mandatory, and if the check fails Sourcegraph will disable all Enterprise features until a successful license check is completed.
+License keys should not be shared across instances of Sourcegraph. If an additional license key is required for something like a dev environment, please [contact customer support](https://about.sourcegraph.com/contact) for an additional license key.
 
-<span class="badge badge-note">Sourcegraph 5.1+</span>
+## FAQ
+
+### What happens if our Sourcegraph license expires?
+
+#### After Sourcegraph 5.3
 
-If Sourcegraph needs to operate in an air-gapped environment without an external internet connection, contact customer support for a special license key.
+Sourcegraph will continue to function as normal, but all users will be signed out of Sourcegraph. Only site administrators will be able to sign into Sourcegraph so that they can enter a new license key.
 
-License keys also need to be unique to a single instance of Sourcegraph. If the same license key is used across multiple instances, subsequent license checks will fail. If multiple license keys are required for dev/staging instances, contact customer support for additional license keys for these instances.
+#### Before Sourcegraph 5.3
 
-## How to update your license key
+Sourcegraph will revert to a free license, and any features that require an enterprise license will stop functioning. This could lead to data loss if some of these features were in use, so be sure to renew your license in advance!
 
-Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](/admin/config/site_config)
+### How can we update our license key?
+
+Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](../config/site_config.md)
```

Comment:
```suggestion
Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](/admin/config/site_config)
```
The links will not work with `.md` extension, and I corrected the path. The parent folder, which is `admin` in this case, should be included. :)

Reply:
Ah gotcha, thanks!